functions.php

```diff
@@ -117,7 +117,8 @@ function twentynineteen_content_width() {
 function twentynineteen_scripts() {
 	wp_enqueue_style( 'twentynineteen-style', get_stylesheet_uri() );
 
-	wp_enqueue_script( 'twentynineteen-skip-link-focus-fix', get_template_directory_uri() . '/js/skip-link-focus-fix.js', array(), '20151215', true );
+	wp_enqueue_script( 'twentynineteen-skip-link-focus-fix', get_template_directory_uri() . '/js/skip-link-focus-fix.js', array(), '20151215', false );
+	wp_script_add_data( 'twentynineteen-skip-link-focus-fix', 'defer', true );
```

Comment:
This will be unnecessary (for `skip-link-focus-fix`) with this change applied: https://github.com/Automattic/_s/pull/1323

See also note on https://github.com/wprig/wprig/pull/139 for why `defer` is not ideal for `skip-link-focus-fix` either.

Reply:
Right, I totally forgot. The functionality still stands though (I expect some JS will be added to this theme at some point.)

Reply:
PR to apply the fix to the blocking script: https://github.com/WordPress/twentynineteen/pull/47